src/main/java/org/openrewrite/java/migrate/lang/var/UseVarForPrimitive.java

```diff
@@ -101,7 +101,7 @@ private J.VariableDeclarations transformToVar(J.VariableDeclarations vd) {
                 return template.apply(getCursor(), vd.getCoordinates().replace(), simpleName, initializer)
                         .withPrefix(vd.getPrefix());
             } else {
-                J.VariableDeclarations result = template.<J.VariableDeclarations>apply(getCursor(), vd.getCoordinates().replace(), simpleName, initializer)
+                J.VariableDeclarations result = template.apply(getCursor(), vd.getCoordinates().replace(), simpleName, initializer)
```

Comment:
These changes trip up the compiler; we should make the recipe back off in these cases.